addons/website_hr_recruitment/views/website_hr_recruitment_templates.xml

```diff
@@ -306,43 +316,43 @@
         <div class="container">
             <div class="row">
                 <div class="text-center col-lg-3 mt16 mb16">
-                    <h4 class="mb0 mt8"><span class="fa fa-2x fa-heart"/>
+                    <h4 class="mb0"><span class="fa fa-2x fa-heart mb8"/>
```

Comment:
It is really broken without this? xml change in stable should be avoided if no real reason.

Reply:
Without that change I find it ugly and the fp request was to prettify the view